### PR TITLE
241 accessioning numbers

### PR DESCRIPTION
**Pull request by cdmo**
#241 

#246 will need to be reviewed first, this branch built on top of that.

### CODE REVIEW

File: spec/lib/traject/psulib_config_spec.rb [hunk 1]
```diff
@@ -4,26 +4,37 @@
...
 
-  describe 'Record with music numeric should have semicolons for all but subfield e' do
-    let(:field) { 'music_numerical_ssm' }
-    let(:music_383) do
-      { '383' => { 'ind1' => '1', 'ind2' => '0', 'subfields' => [{ 'b' => 'op. 36' },
-                                                                 { 'b' => 'op. 86' },
-                                                                 { 'b' => 'op. 35' },
-                                                                 { 'e' => 'Bach' },
-                                                                 { 'e' => 'Motzart' }] } }
+  describe 'id' do
```
Comment (banukutlu) [inline spec/lib/traject/psulib_config_spec.rb:15]: id and music numeric specs just re-ordered right?

Comment (cdmo) [reply spec/lib/traject/psulib_config_spec.rb:15]: yeah that's what I meant in the commit message, sorry, not clear.

I also have to say, as a bigger topic, do we want the new spec I added for this? I think the answer is "yes" but we also have dozens of straightforward `extract_marc` fields un-tested.... This is relevant for work required in https://github.com/psu-libraries/psulib_traject/issues/244 too.  Again my leaning is that "yes" they need tests just for clarity. But I dont know that I want to add tests for all of our fields. How much value do we get for these tests?

Comment (banukutlu) [reply spec/lib/traject/psulib_config_spec.rb:15]: I think we initially decided not to add specs for just straightforward extract_marc fields, and just add ones for that involves additional logic. 
Saying that, I think it is OK even nice to have a spec for a newly added field, kinda gives a piece of mind. 

If we wanna consider about adding a spec for those other fields too, I agree that might be a work that might not hold as much value in the end but maybe we can add several fixtures that captures all the fields and somehow find a way to just test result of the whole import, kinda like a feature specish.

**Review by banukutlu — APPROVED** [review]
👍

File: spec/lib/traject/psulib_config_spec.rb (hunk 1 above)
Comment (cdmo) [reply spec/lib/traject/psulib_config_spec.rb:15]: think we're on the same page 👍

**Pull request merged**